### PR TITLE
Modified README for docker

### PR DESCRIPTION
**Pull request by melvyniandrag**
Please accept my PR! I'm trying to get a hacktoberfest tshirt and I'm new to opensource, just trying to get my feet wet with how PRs work on github.

I was skimming alot of information about docker and tensorflow online and when I came to the section of the readme I changed I had to stop and read carefully for a second to think about what was happening there, when I should have just skipped over it and kept going. Just added a comment that made it even more explicit that this code snipped is outdated and not recommended.

### CODE REVIEW

**Comment by melvyniandrag** [comment]
I signed it!

**Review by caisq — APPROVED** [review]

File: tensorflow/tools/docker/README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Note: If you would have a problem running nvidia-docker you may try the old meth
 we have used. But it is not recommended. If you find a bug in nvidia-docker, please report
 it there and try using nvidia-docker as described above.
 
+    $ # The old, not recommended way to run docker with gpu support:
```
Comment (frankchn) [inline tensorflow/tools/docker/README.md]: Can you say something like `$ # The old method to run docker with GPU support`

Comment (frankchn) [reply tensorflow/tools/docker/README.md]: Actually, on second thought this is fine too.

**Review by frankchn — APPROVED** [review]

**Comment by frankchn** [comment]
Jenkins, test this please.

**Pull request merged**